src/main/java/frc/robot/Robot.java

```diff
@@ -5,7 +5,10 @@
 package frc.robot;
 
 import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
 import edu.wpi.first.wpilibj2.command.*;
```

Comment:
This line does not import anything into this class and I recommend deleting this line.